docs/pages/Cardinal/API-Reference/World.mdx

```diff
@@ -0,0 +1,529 @@
+import { Callout } from 'nextra/components'
+
+# World
+
+The `World` object is the manager of everything happening in the game. It deals with registering components, systems, transactions, and reads, as well as ticking the game simulation forward. `World` objects use [Redis](https://redis.io/) to store the game's state.
+
+
+```go
+package main
+
+import (
+    "log"
+
+    "pkg.world.dev/world-engine/cardinal/ecs"
+)
+
+func main() {
+    redisAddr := os.Getenv("redis_address")
+    redisStore := storage.NewRedisStorage(storage.Options{Addr: redisAddr}, "my-world")
+
+    worldStore := storage.NewWorldStorage(&redisStore)
+
+    world, err := ecs.NewWorld(
+		worldStore,
+		ecs.WithNamespace("my-world"),
+		ecs.WithReceiptHistorySize(10),
+	)
+    if err != nil {
+        log.Fatal(err)
+    }
+}
+```
+
+## NewWorld
+
+`NewWorld` creates a new `World` object.
+
+```go
+func NewWorld(s storage.WorldStorage, opts ...Option) (*World, error)
+```
+
+### Parameters
+| Parameter | Type                        | Description                                               |
+|-----------|-----------------------------|-----------------------------------------------------------|
+| s         | storage.WorldStorage       | The storage backend to be used for the world.             |
+| opts      | ...Option                  | Optional functional options for configuring the world.    |
+
+### Return Values
+
+| Type  | Description                                      |
+|--------------|--------------------------------------------------|
+| *World       | A pointer to the newly created World instance.  |
+| error        | An error indicating any issues during creation. |
+
+### Options
+
+#### WithAdapter
+
+The `WithAdapater` world option enables `World` objects to read transactions from the EVM base shard, and rebuild game state from those transactions.
```

Comment:
I think we can make this more

The `WithAdapter` option allows Cardinal to post and recover transaction calldata from the sequencer (EVM base shard). When running Cardinal as a rollup in production, `WithAdapter` must be used.

Reply:
Also, typo. `WithAdapater` -> `WithAdapter`

Reply:
This option does not enable the world to post any data, only read. Posting is the job of the server

Reply:
[docs: add note that WithAdapter must be used in prod](https://github.com/Argus-Labs/world-engine/pull/200/commits/b9cd972f478ec82a461969aebef779775f6da269) added note for prod

Reply:
@technicallyty we should definitely create a ticket on linear to consolidate that into the same thing. i think we had this convo a while back.

Reply:
its still an open ticket - https://linear.app/arguslabs/issue/CAR-110/have-world-automatically-setup-the-transaction-handler